gui.go

```diff
@@ -611,6 +611,8 @@ func (g *Gui) WhitelistKeybinding(k Key) error {
 // typed Key or rune.
 func getKey(key interface{}) (Key, rune, error) {
 	switch t := key.(type) {
+	case nil:
```

Comment:
can we add a comment explaining what a nil value means?

Reply:
I have added a comment to this line.